helm/flux-app/values.yaml

```diff
@@ -13,6 +13,10 @@ images:
   imageReflectorController:
     image: giantswarm/fluxcd-image-reflector-controller
 
+# allows to change cluster related configurations
+cluster:
+  domain: cluster.local
```

Comment:
Why not hardcode to the value we need right now
Saves us some time

Reply:
As default we should just leave it empty? Fits on different clusters then without changes.